Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -283,13 +283,17 @@ protected function setUp(): void
             self::$currestTestCaseClass = $currentTestCaseClass;
         }
 
+        // sqlite db path preparation
+        $dbPathSqlite = dirname($this->instancePath) . '/functional-sqlite-dbs/test_' . $this->identifier . '.sqlite';
+        $dbPathSqliteEmpty = dirname($this->instancePath) . '/functional-sqlite-dbs/test_' . $this->identifier . '.empty.sqlite';
+
         if (!$isFirstTest) {
             // Reusing an existing instance. This typically happens for the second, third, ... test
             // in a test case, so environment is set up only once per test case.
             GeneralUtility::purgeInstances();
             $this->container = $testbase->setUpBasicTypo3Bootstrap($this->instancePath);
             if ($this->initializeDatabase) {
-                $testbase->initializeTestDatabaseAndTruncateTables();
+                $testbase->initializeTestDatabaseAndTruncateTables($dbPathSqlite, $dbPathSqliteEmpty);
             }
             $testbase->loadExtensionTables();
         } else {
```

Comment:
Should we not remove old sqlite database path on first instance creation, like it is done for instance files, since it is now seperated and coverd from the $this->instance_path anymore ? (Even if it is eventually in a ram disk ?)